Description: A patch for output-pdf.c before 0.31.1-16
 This patch has been created to represent all changes was made direct
 to the package before implement dpkg-src 3.0(quilt) format, which has 
 a patch system.
 .
 This help to eliminate redundant closepaths in pdf output(Thanks to 
 Paul Sladen) and switch to size_t for handle 64bits. 
Author: Edgar Antonio Palma de la Cruz <xbytemx@gmail.com>
Last-Update: 2011-08-07

--- autotrace-0.31.1.orig/output-pdf.c
+++ autotrace-0.31.1/output-pdf.c
@@ -201,16 +201,16 @@ static int output_pdf_tailor(FILE* pdf_f
   tmp += (strlen (temp));
   sprintf(temp, "%d", ury);
   tmp += (strlen (temp));
-  OUT1     ("%010d 00000 n \n", tmp);
-  sprintf(temp, "%d", length);
+  OUT1     ("%010"SIZET_FORMAT"u 00000 n \n", tmp);
+  sprintf(temp, "%"SIZET_FORMAT"u", length);
   tmp += 50 + length + strlen(temp);
-  OUT1     ("%010d 00000 n \n", tmp);
+  OUT1     ("%010"SIZET_FORMAT"u 00000 n \n", tmp);
   OUT_LINE ("trailer");
   OUT_LINE ("   << /Size 7");
   OUT_LINE ("      /Root 1 0 R");
   OUT_LINE ("   >>");
   OUT_LINE ("startxref");
-  OUT1 ("%d\n", tmp + 25);
+  OUT1 ("%"SIZET_FORMAT"u\n", tmp + 25);
   OUT_LINE ("%%EOF");
 
   return 0;
@@ -243,7 +243,8 @@ out_splines (FILE *pdf_file, spline_list
           if (this_list > 0)
             {
               SOUT_LINE ((shape.centerline || list.open) ? "S" : "f");
-              SOUT_LINE("h");
+              /* In PDF a Stroke (S) or fill (f) causes an implicit closepath (h) -Paul Sladen */
+              /* SOUT_LINE("h"); */
             }
           SOUT4 ("%.3f %.3f %.3f %s\n", (double) list.color.r / 255.0,
             (double) list.color.g / 255.0, (double) list.color.b / 255.0,
@@ -270,7 +271,7 @@ out_splines (FILE *pdf_file, spline_list
     SOUT_LINE ((shape.centerline || list.open) ? "S" : "f");
 
   OUT_LINE ("5 0 obj");
-  OUT1 ("   << /Length %d >>\n", *length);
+  OUT1 ("   << /Length %"SIZET_FORMAT"u >>\n", *length);
   OUT_LINE ("stream");
 
   last_color.r = 0;
@@ -291,7 +292,8 @@ out_splines (FILE *pdf_file, spline_list
           if (this_list > 0)
             {
               OUT_LINE ((shape.centerline || list.open) ? "S" : "f");
-              OUT_LINE("h");
+              /* In PDF a Stroke (S) or fill (f) causes an implicit closepath (h) -Paul Sladen */
+              /* OUT_LINE("h"); */
             }
           OUT4 ("%.3f %.3f %.3f %s\n", (double) list.color.r / 255.0,
             (double) list.color.g / 255.0, (double) list.color.b / 255.0,
